nixnet/errors.py

```diff
@@ -21,8 +21,8 @@ def __init__(self, message, error_code):
         """Initialize error.
 
         Args:
-            message (string): Specifies the error message.
-            error_code (int): Specifies the NI-XNET error code.
+            message: A string specifing the error message.
+            error_code: An integer specifing the NI-XNET error code.
```

Comment:
This will be a big change so I'll do it in a separate PR. For now I've modified types, errors, etc. to match majority of the docstrings in the project.